Sources/NIO/ContiguousCollection.swift

```diff
@@ -12,12 +12,6 @@
 //
 //===----------------------------------------------------------------------===//
 
-/// A `Collection` that is contiguously laid out in memory and can therefore be duplicated using `memcpy`.
-public protocol ContiguousCollection: Collection {
-    @inlinable
-    func withUnsafeBytes<R>(_ body: (UnsafeRawBufferPointer) throws -> R) rethrows -> R
-}
-
 extension StaticString: Collection {
```

Comment:
Do we need the StaticString conformance here?

Reply:
good catch!